shape_pytest.py

```diff
@@ -41,3 +44,44 @@ def test_circle():
     assert radius8.perimeter() == 50.27
     assert radius5.diameter() == 10
     assert radius8.diameter() == 16
+
+
+def test_triangle():
+    """
+    Defines tests on some specific triangle objects.
+    """
+    a3b3c3 = Triangle(3, 3, 3)
```

Comment:
If you decided to implement the new constructor suggestions below:

`a3b3c3 = Triangle({'a': 3, 'b': 3, 'c': 3)`

It isn't better or worse, just different.